cob_light/ros/src/cob_light.cpp

```diff
@@ -191,12 +195,12 @@ class LightControl
     //Subscribe to LightController Command Topic
     _sub = _nh.subscribe("command", 1, &LightControl::topicCallback, this);
 
-    //Subscribe to LightController Command Topic
-    _sub_mode = _nh.subscribe("light", 1, &LightControl::topicCallbackMode, this);
-
```

Comment:
because i've removed the possibility to switch modes via topic. This functionally was deprecated for a long time